Dockerfile.assisted-installer-build

```
@@ -2,7 +2,7 @@ FROM registry.ci.openshift.org/openshift/release:golang-1.17
 ENV GO111MODULE=on
 ENV GOFLAGS=""
 
-COPY --from=quay.io/app-sre/golangci-lint:v1.37.1 /usr/bin/golangci-lint /usr/bin/golangci-lint
+RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/bin v1.37.1
```

maybe we should install from source, this will ensure to work if we user different type of conainers
 ```suggestion
RUN go install github.com/golangci/golangci-lint/cmd/golangci-lint@v1.37.1
```